ambari-server/src/main/resources/stacks/stack_advisor.py

```diff
@@ -1304,8 +1304,9 @@ def getConfigurationClusterSummary(self, servicesList, hosts, components, servic
 
 
     '''containers = max(3, min (2*cores,min (1.8*DISKS,(Total available RAM) / MIN_CONTAINER_SIZE))))'''
+    core_multiplier = 2 if not cpu_only_mode else 1
```

Comment:
@vbrodetskyi @rlevas i believe that name for the variable is wrong, until it would be described.  I barely can understands why this is needed and why we can't just add:
```
 def getConfigurationClusterSummary(self, servicesList, hosts, components, services, cpu_multiplier = 2):
  ...
```